src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -90,25 +90,92 @@ public T capture() {
      * If verified method was called multiple times then this method it returns the latest captured value.
      * <p>
      * See examples in javadoc for {@link ArgumentCaptor} class.
-     * 
+     *
      * @return captured argument value
+     * @deprecated please use {@link #getLastValue()}
      */
+    @Deprecated
     public T getValue() {
         return this.capturingMatcher.getLastValue();
     }
 
     /**
+     * Returns the captured value of the argument. When capturing varargs use {@link #getAllValues()}.
+     * <p>
+     * If verified method was called multiple times then this method it returns the latest captured value.
+     * <p>
+     * See examples in javadoc for {@link ArgumentCaptor} class.
+     *
+     * @return captured argument value
+     */
+    public T getLastValue() {
+        return this.capturingMatcher.getLastValue();
+    }
+
+    /**
+     * Returns the captured value of the argument. When capturing varargs use {@link #getAllValues()}.
+     * <p>
+     * If verified method was called at least once then method it returns the first captured value.
+     * <p>
+     * See examples in javadoc for {@link ArgumentCaptor} class.
+     *
+     * @return captured argument value
+     */
+    public T getFirstValue() {
+        return getAllValues().get(0);
+    }
+
+    /**
+     * Returns the captured value of the argument. When capturing varargs use {@link #getAllValues()}.
+     * <p>
+     * If verified method was called at least twice then method it returns the second captured value.
+     * <p>
+     * See examples in javadoc for {@link ArgumentCaptor} class.
+     *
+     * @return captured argument value
+     */
+    public T getSecondValue() {
+        return getAllValues().get(1);
+    }
+
+    /**
+     * Returns the captured value of the argument. When capturing varargs use {@link #getAllValues()}.
+     * <p>
+     * If verified method was called at least thrice then method it returns the third captured value.
+     * <p>
+     * See examples in javadoc for {@link ArgumentCaptor} class.
+     *
+     * @return captured argument value
+     */
+    public T getThirdValue() {
+        return getAllValues().get(2);
+    }
+
+    /**
+     * Returns the captured value of the argument. When capturing varargs use {@link #getAllValues()}.
+     * <p>
+     * If verified method was called multiple times then this method it returns the latest captured value at the given index.
+     * <p>
+     * See examples in javadoc for {@link ArgumentCaptor} class.
+     *
+     * @return captured argument value
+     */
+    public T getValueAt(int index) {
```

Comment:
I am still convinced the At suffix is better, but since I voiced my opinion already let's wait for the response of the other @mockito/developers